website/templates/emails/conference_submitted.txt.mako

```diff
@@ -21,9 +21,9 @@ Get more from the OSF by enhancing your project with the following:
 * Collaborators/contributors to the submission
 * Charts, graphs, and data that didn't make it onto the submission
 * Links to related publications or reference lists
-* Connecting other accounts, like Dropbox, Google Drive, GitHub, figshare and Mendeley via add-on integration. Learn more and read the full list of available add-ons: http://osf.io/getting-started/#addons
+* Connecting other accounts, like Dropbox, Google Drive, GitHub, figshare and Mendeley via add-on integration. Learn more and read the full list of available add-ons: http://osf.io/getting-started/email#addons
```

Comment:
As this PR currently stands, this link will fold down into the base `/getting-started/email` redirect, but [OSF-4998](https://openscience.atlassian.net/browse/OSF-4998) calls for a second redirect specifically for add-ons. Either we should implement the redirect in JavaScript so we can switch on the fragment as originally specified, or we should drop back to `/getting-started/email-addons` and implement the second redirect in Python instead.